test/metabase/lib/test_util/generators/filters.cljc

```diff
@@ -0,0 +1,278 @@
+(ns metabase.lib.test-util.generators.filters
+  (:require
+   [metabase.lib.core :as lib]
+   [metabase.lib.options :as lib.options]
+   [metabase.lib.test-util.generators.util :as gen.u]
+   [metabase.lib.types.isa :as lib.types.isa]
+   [metabase.util.time :as u.time]))
+
+;; Filter values =================================================================================
+(defn- gen-int []
+  (- (rand-int 2000000) 1000000))
```

Comment:
Not that these one-liners are hurting anything, but just wondering if `test.check` has generators for simple types like this that would be useful / reusable for our purposes. (I know nothing about `test.check` other than its existence, but reading the docs is in my near-term TODO list).

Reply:
I actually tried to use them first, but found that they don't fit into this model very well. They have "shrinking" logic that aims to start from the smallest values, and it just doesn't fit into this random walk model at all.